src/libs/rust/libc/resources/headers/kdb.h

```diff
@@ -0,0 +1,155 @@
+#include <stdint.h>
+
+enum {
+	ELEKTRA_KEY_KEY_VALUE=1<<1,
```

Comment:
```suggestion
	ELEKTRA_KEY_VALUE=1<<1,
```

This was an error in #4242. It was pointed out but not fixed in #4242. It also effects all other fields of this enum.